packages/ra-ui-materialui/src/detail/Tab.tsx

```diff
@@ -126,7 +129,7 @@ Tab.propTypes = {
     icon: PropTypes.element,
     label: PropTypes.string.isRequired,
     path: PropTypes.string,
-    value: PropTypes.string,
+    value: PropTypes.any,
```

Comment:
Why did you change that?

Reply:
Because we can use path or indexes which are numbers. I should probably be more specific though. However, MUI value for the `Tabs` is a TS any